packages/jest-runtime/src/should_instrument.js

```diff
@@ -37,7 +37,8 @@ export default function shouldInstrument(
 
   if (
     config.testRegex &&
-    config.testRegex.some(regex => regex.test(filename))
+    config.testRegex.length &&
```

Comment:
Is this guard necessary?

Reply:
Normalize should still guarantee it's an array, imo

Reply:
Nope! Bad copy & paste.